torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1979,7 +1964,24 @@ void ONNXShapeTypeInference(
     for (auto output : clone_node->outputs()) {
       n_graph->registerOutput(output);
     }
-
+    // Use outputs_map for original PyTorch output name to corresponding ONNX output name
```

Comment:
A comment like this belongs above the variable declaration. Even better: choose a more descriptive variable name so the comment is not necessary :-)

Maybe: `torch_to_onnx_output_name`

Reply:
Good point. I will use torch_to_onnx_output_name.